.github/workflows/build.yml

```diff
@@ -1,20 +1,10 @@
-name: Build Code Samples 
+name: Build Code Samples
 
 on:
   push:
-    branches: [release/3.2]
-    paths:
-      - 'modules/test/*'
-      - 'modules/Makefile'
-      - '**/*.java'
-      - '**/pom.xml'
+    branches: [ release/3.* ]
   pull_request:
```

Comment:
is this line needed? seems not to have any child keys

Reply:
(same in at least one other .yml file, I won't repeat the comment this time 😄 )

Reply:
I was trying to see if not specifying the branch would still work, which it does, I don't think it will allow me to do `pull_request` without the `:` at the end.

Alternatively, I could just add `branches: [ release/3.* ]` in there too to be explicit. Probably what I *should* do, I'll update